libevmasm/ControlFlowGraph.cpp

```diff
@@ -275,7 +275,7 @@ void ControlFlowGraph::gatherKnowledge()
 			//@todo in the case of JUMPI, add knowledge about the condition to the state
 			// (for both values of the condition)
 			std::set<u256> tags = state->tagsInExpression(
-				state->stackElement(state->stackHeight(), langutil::SourceLocation{})
+				state->stackElement(state->stackHeight(), yul::DebugData::create())
```

Comment:
That's what I meant in the other comment about the static default source location:
If we create empty debug data in cases like this, we might as well *always* guarantee debug data being present in assembly items. Otherwise we could just pass a null pointer here instead